docs/orchestrating-elastic-stack-applications/elasticsearch/orchestration.asciidoc

```diff
@@ -54,88 +62,118 @@ spec:
+Based on how Kubernetes and StatefulSets operate, ECK orchestration has the following limitations:
 
-* Storage requirements (including volume size) of an existing `NodeSet` cannot be updated. link:https://github.com/kubernetes/enhancements/issues/661[StatefulSet volumes expansion is not available in Kubernetes yet]. To upgrade the storage size, you can create a new `NodeSet`, or rename an existing one. Renaming a `NodeSet` automatically creates a new `StatefulSet` with the specified storage size. The original `StatefulSet` is removed once the Elasticsearch data is migrated to the nodes of the new `StatefulSet`.
+* Storage requirements (including volume size) of an existing NodeSet cannot be updated. link:https://github.com/kubernetes/enhancements/issues/661[StatefulSet volumes expansion] is not available in Kubernetes yet. To upgrade the storage size, you can create a new NodeSet, or rename an existing one. Renaming a NodeSet automatically creates a new StatefulSet with the specified storage size. The original StatefulSet is removed once the Elasticsearch data is migrated to the nodes of the new StatefulSet.
 
-* Cluster availability is not be guaranteed in the following cases:
+* Cluster availability is not guaranteed in the following cases:
 
 ** During the rolling upgrade of single-node clusters
 
 ** For clusters that have indices with no replicas
 
-If an Elasticsearch node holds the only copy of a shard, this shard becomes unavailable while the node is upgraded. Clusters with more than one node and at least one replica per index are considered best practice.
+If an Elasticsearch node holds the only copy of a shard, this shard becomes unavailable while the node is upgraded. Clusters with more than one node and at least one replica per index are recommended.
 
-* Elasticsearch `Pods` may stay `Pending` during a rolling upgrade if the Kubernetes scheduler cannot re-schedule them back. This is especially important when using local `PersistentVolumes`. If the Kubernetes node bound to a local `PersistentVolume` does not have enough capacity to host an upgraded `Pod` which was temporarily removed, that `Pod` will stay Pending.
+* Elasticsearch Pods may stay `Pending` during a rolling upgrade if the Kubernetes scheduler cannot re-schedule them back. This is especially important when using local PersistentVolumes. If the Kubernetes node bound to a local PersistentVolume does not have enough capacity to host an upgraded Pod which was temporarily removed, that Pod will stay `Pending`.
 
 * Rolling upgrades can make progress if the Elasticsearch cluster health is green. ECK will also make progress if the cluster health is yellow under the following conditions:
-** A cluster version upgrade is in progress and some `Pods` are not up to date
-** There are no initializing or relocating shards
+** A cluster version upgrade is in progress and some Pods are not up to date.
```

Comment:
```suggestion
** A cluster version upgrade is in progress and some Pods are not up-to-date.
```

Reply:
Hyphenate it only when used as an adjective => an up-to-date Pod.